test/end-to-end/auth.js

```diff
@@ -147,6 +147,8 @@ describe("Login endpoint test", () => {
 
             expect(res.status).toBe(HTTPStatus.OK);
             expect(res.body).toHaveProperty("data.email", test_user.email);
+            expect(res.body).toHaveProperty("data.isAdmin", true);
```

Comment:
Are there not other tests in which it would make sense to test that the users are specifically not admins? (Like a non-admin user using this endpoint?)

Additionally, for the following line a similar question: is there any test that has a user with a company and could be used to test this already?

Reply:
That feature is not yet being used (we only have logic specific for admins so far in the UI), that's why I did not work on that yet, so that we can move faster. They should be added once we start having logic for company users login. Created #86 for that

Reply:
Thanks for creating the issue, this should be fine then